tsconfig.json

```diff
@@ -5,7 +5,7 @@
         "sourceMap": true,
         "moduleResolution": "classic",
         "lib": [
-            "es7",
+            "es2018",
```

Comment:
this was already needed for:
https://github.com/mebjas/html5-qrcode/blob/a14ca5e3c9cbd308944062c1965c7cb172e6482f/src/html5-qrcode.ts#L1033